engine/commands/cortex_upd_cmd.h

```diff
@@ -11,7 +10,7 @@ namespace commands {
 #ifndef CORTEX_VARIANT
 #define CORTEX_VARIANT file_manager_utils::kProdVariant
 #endif
-constexpr const auto kNightlyHost = "https://delta.jan.ai";
+constexpr const auto kNightlyHost = "delta.jan.ai";
```

Comment:
Should these be pulled out into a `.cortexrc`? It seems very fragile to hard code a lot of these? cc @vansangpfiev @nguyenhoangthuan99

Reply:
`.cortexrc` should be for things that are configurable for user? I doubt it's a suitable place.
And, IMO, we are not hard-coded here. We are defining compile time constants.